templates/app.yaml

```diff
@@ -246,6 +246,10 @@ Resources:
         - Namespace: 'aws:elasticbeanstalk:application:environment'
           OptionName: TEAM_TO_ROLE_ARN_MAP
           Value: "ssm::/service-catalog/TeamToRoleArnMap"
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: MARKETPLACE_ID_DYNAMO_TABLE_NAME
+          Value: !ImportValue
```

Comment:
hmm, sorry I forgot we save all configs for the login app to the SSM.  I think we should follow the same pattern here instead of using a direct `!ImportValue`.  

So please add the param and value here instead.. https://github.com/Sage-Bionetworks/synapse-login-aws-infra/blob/develop/config/develop/synapse-login-scipooldev-params.yaml

and change this to  `Value:  "ssm::/service-catalog/MarketplaceIdDynamoTableName"` so the login app can get the value from the SSM.